tmt/checks/__init__.py

```diff
@@ -100,6 +114,12 @@ class Check(
         default=True,
         is_flag=True,
         help='Whether the check is enabled or not.')
+    result: CheckResultInterpret = field(
+        default=CheckResultInterpret.RESPECT,
+        help='How to interpret the check result.',
+        serialize=lambda result: result.value,
+        unserialize=CheckResultInterpret.from_spec
+        )
```

Comment:
Just a small nitpick, move the `)` right after `unserialize=CheckResultInterpret.from_spec` to avoid an unnecessary line.

Reply:
addressed in https://github.com/teemtee/tmt/pull/3239/commits/345bde830384b610d0d4bccd395b7f0a00aa5b26